src/shared/install.h

```diff
@@ -32,6 +32,7 @@ enum {
         UNIT_FILE_UNLINK,
         UNIT_FILE_IS_MASKED,
         UNIT_FILE_IS_DANGLING,
+        UNIT_FILE_IS_UNKNOWN_TARGET,
```

Comment:
i don't like the name of this. the specified unit can be of any type, not just target units

Reply:
maybe: "vacant"? in the sense of "squatting a vacant lot"

Reply:
Well, it's supposed to mean dependency target, not a unit type...